configs/sst_conversions.yaml

```diff
@@ -0,0 +1,30 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: RHC Worker Bash
+  description: Remote Host Configuration (rhc) worker for executing bash scripts on hosts managed by Red Hat Insights.
+  maintainer: sst_conversions
+
+  packages:
+    - rhc-worker-bash
+  
+  labels:     
+  - c9s # TODO(r0x0d): Verify if this is the correct label for RHEL 7
+
+  arch_packages:    
+    x86_64:
+    - rhc-worker-bash
+
+  # Add packages to the workload that don't exist (yet) in the repositories.
+  package_placeholders:
+    - srpm_name: rhc-worker-bash  # SRPM name (mandatory)
+      build_dependencies:   # build dependencies (RPM names) of this SRPM (optional)
+        - golang
+      limit_arches:         # can be left empty for all arches (optional)
+        - x86_64
```

Comment:
Why is this x86_64 only?  The rhc package is offered on all architectures.

Reply:
Not exactly sure if the worker will have support for all other architectures. For now, I added x86_64 as a placeholder. Will confirm with other people to make sure what architectures we will be supporting.